bundles/cif-connector-graphql/src/test/java/com/adobe/cq/commerce/graphql/magento/GraphqlDataServiceImplTest.java

```diff
@@ -254,6 +254,45 @@ public void testGetProductBySkuError() throws Exception {
         assertNotNull(exception);
     }
 
+    @Test
+    public void testGetCategoryByIdError() throws Exception {
+        Utils.setupHttpResponse("magento-graphql-error.json", httpClient, HttpStatus.SC_OK);
+        Exception exception = null;
```

Comment:
A nicer way to test for exceptions is to use the `ExpectedException` rule as described for example in https://www.baeldung.com/junit-assert-exception#junit-4.

Reply:
@herzog31 , indeed. I've fixed it.